src/components/app.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+import Header from './header';
+
+export default class App extends Component {
+  render() {
+    return (
+    <div>
```

Comment:
nit: spacing (should be indented, at least from what I see in the react getting started examples)